### ISSUE TITLE
Close Github Discussions

### ISSUE THREAD

**Issue by camilaibs** [backstage/backstage#20985]
Opening this issue for further team discussion:

> We should probably look at closing the discussions down then, same for the discussions in discord too if we prefer that.

_Originally posted by @benjdlambert in https://github.com/backstage/backstage/pull/20837#pullrequestreview-1704591443_

**Comment by tudi2d** [comment]
Yeah. Let's make sure to direct people over who recently openeded discussions to make it less frustrating.

It would be interesting to understand when people are posting in GitHub - if there are common themes/topics. Maybe we can find a low effort solution like improving the FAQ for some topics. Also I can see that it is more cumbersome to join a Discord community - especially if you are not used to Discord. Even though it's not perfect, but maybe we can have a GitHub Issue Template for less technical discussions like adoption or usage of specific plugins.

**Comment by Rugvip** [comment]
For now we have tried out disabling GitHub Discussions. We're very much looking for feedback on this so if you think this is a mistake and would like to keep GitHub Discussions enabled, please chime in here.

The primary reasons for this is that we don't currently have a structure in place for how to respond to discussions, and there's significant overlap with issues and the Discord support channel. As we're moving towards distributing the maintenance of the project more and more it's important to have clear ways for how to split work across project areas, while at the same time not having too many small sections to cover for each area. There's also significantly more traffic on Discord, and we feel it's best to direct people asking questions over there instead so that it's more likely that their message gets a response.

The main upside of GitHub Discussions was the structured Q&A format with accepted answers that is also more searchable than Discord discussions. The project is still moving fast though, so getting help around more temporary issues like upgrades or feature gaps is arguably better via Discord, while the ideally the best place to search for up-to-date solutions to common problems is backstage.io and other sources of documentation.

**Comment by OrkoHunter** [comment]
Was just looking for a discussion thread and found the links from Google results are broken 😅 

The biggest downside of using Discord as a knowledge-base is the information not being public. Several enterprises do not allow creating Discord accounts and even accessing it on their work computers. GitHub is mostly allowed everywhere. Although Backstage Discord definitely has a lot of activity, I believe we also need an open and public platform for building Q&A knowledge base too.

Perhaps allow a "Question" as an Issue type too? Or use issues on backstage/community for all FAQs.
